backstop/src/storage.rs

```diff
@@ -41,8 +41,8 @@ pub struct UserEmissionData {
 #[derive(Clone)]
 #[contracttype]
 pub struct PoolUserKey {
-    pool: Address,
-    user: Address,
+    pub pool: Address,
+    pub user: Address,
```

Comment:
not sure I see why these needed to be public?

Reply:
I needed them public for tests and forgot to remove whoops